README.md

```diff
@@ -1,13 +1,11 @@
 # Project React Native App 📱
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week we were to build an app using React Native. We should be using either a phone sensor like the accelerometer, an api-fetch or React Navigation. The app should be published using Expo.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://expo.dev/@mathilda_karlsson/project-react-native-app?serviceType=classic&distribution=expo-go
```

Comment:
Yeay! I managed to open your expo-app. Have tried on others but failed. So I got the opportunity to test your app and the functionality was there and so was the simple styling. You have added different features to your app which gives it a very user friendly approach!